modules/ROOT/pages/clauses/load-csv.adoc

```diff
@@ -286,116 +450,281 @@ CALL {
 ----
-LOAD CSV FROM 'file:///artists.csv' AS line
-RETURN linenumber() AS number, line
+LOAD CSV FROM 'https://data.neo4j.com/importing-cypher/acted_in.csv' AS row
+MATCH (p:Person {tmdbId: toInteger(row.person_tmdbId)})
+MATCH (m:Movie {movieId: toInteger(row.movieId)})
+MERGE (p)-[r:ACTED_IN]->(m)
+SET r.role = row.role
 ----
 
 .Result
 [role="queryresult"]
 ----
-+---------------------------------------+
-| number | line                         |
-+---------------------------------------+
-| 1      | ["1","ABBA","1992"]          |
-| 2      | ["2","Roxette","1986"]       |
-| 3      | ["3","Europe","1979"]        |
-| 4      | ["4","The Cardigans","1992"] |
-+---------------------------------------+
-4 rows
++-------------------+
+| No data returned. |
++-------------------+
+Relationships created: 372
+Properties set: 372
 ----
 
 
-[[load-csv-using-file-with-load-csv]]
-== Using `file()` with LOAD CSV
-For certain scenarios, like debugging a problem with a csv file, it may be useful to get the absolute path of the file that `LOAD CSV` is operating on.
-The `file()` function provides exactly that or `null` if called without a `LOAD CSV` context.
+[[load-csv-best-practices]]
+== Best practices
 
-.artists.csv
-[source, csv, role="noheaders", filename="artists.csv"]
+This section summarizes some best practices while using `LOAD CSV`.
+
+
+[[load-csv-create-constraints]]
+=== Create CONSTRAINTS
+
+The CSV files _persons.csv_ and _movies.csv_ processed in <<load-csv-type-cast-csv-data>>, <<load-csv-split-list-values>> and <<load-csv-create-relationships>> both contain IDs for the created nodes.
+They uniquely identify a person or a movie node but so far there is no check if they are truly unique.
+Neo4j's concept of constraints is a way of enforcing uniqueness.
+
+To create link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/constraints/examples/#constraints-examples-node-uniqueness[node property uniqueness constraints] for the two IDs:
+
+.Query
+[source, cypher]
 ----
-1,ABBA,1992
-2,Roxette,1986
-3,Europe,1979
-4,The Cardigans,1992
+CREATE CONSTRAINT Person_tmdbId IF NOT EXISTS
+FOR (p:Person)
+REQUIRE p.tmdbId IS UNIQUE
+
+CREATE CONSTRAINT Movie_movieId IF NOT EXISTS
+FOR (m:Movie)
+REQUIRE m.movieId IS UNIQUE
+----
+
+.Result
+[role="queryresult"]
 ----
++-------------------+
+| No data returned. |
++-------------------+
+Added 2 constraints.
+----
+
+Now that there are uniqueness constraints, trying to create a person node with an existing `tmdbId` or a movie node with an existing `movieId` raises an error and doesn't create the node.
+
+Note that creating constraints after importing data is risky, since the creation of a constraint fails if there are nodes or relationship that would violate the constraint, see link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/constraints/examples/#constraints-fail-to-create-a-uniqueness-constraint-due-to-conflicting-nodes[Creating a constraint when there exist conflicting nodes will fail].
```

Comment:
```suggestion
Note that creating constraints after importing data is not recommended, since the creation of a constraint fails if there are nodes or relationship that would violate the constraint, see link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/constraints/examples/#constraints-fail-to-create-a-uniqueness-constraint-due-to-conflicting-nodes[Creating a constraint when there exist conflicting nodes will fail].
```